### PR TITLE
Composer

### PR DESCRIPTION
**Pull request by Artistan**
update fore deprecated Symfony\Component\ClassLoader\ClassMapGenerator
using Composer\Autoload\ClassMapGenerator

### CODE REVIEW

File: src/Console/ModelsCommand.php [hunk 1]
```diff
@@ -634,6 +634,8 @@ protected function createPhpDocs($class)
             $phpdoc->appendTag($tag);
         }
 
+        // this only needs to be on the Eloquent/Model since laravel/framework v5.4.29
+        // but it does not hurt anything having it here...
```
Comment (barryvdh) [inline src/Console/ModelsCommand.php]: Can you remove this comment?

Comment (Artistan) [inline src/Console/ModelsCommand.php]: @barryvdh yep, done.

**Comment by Artistan** [comment]
@barryvdh - bump :)

**Comment by mfn** [comment]
This PR can be closed, this happened already via https://github.com/barryvdh/laravel-ide-helper/pull/660

**Pull request closed without merging**